src/Skills/Components/LightningCannon.gd

```diff
@@ -0,0 +1,6 @@
+extends "res://Skills/Core/Skill.gd"
+
+func _init():
+	type = TYPES.TARGETED
+	signal_name = "lightning_cannon"
+	icon = preload("res://Assets/skills/stunpistol.png")
```

Comment:
This seems incorrect.

Reply:
Yep, need to do artwork to actually reference it here. As you know, not my strong suit. :/